parts/linux/cloud-init/artifacts/cse_config.sh

```diff
@@ -374,6 +381,25 @@ ensureDHCPv6() {
     retrycmd_if_failure 120 5 25 modprobe ip6_tables || exit $ERR_MODPROBE_FAIL
 }
 
+ensureKubeCAFile() {
+    KUBE_CA_FILE="/etc/kubernetes/certs/ca.crt"
+    mkdir -p "$(dirname "${KUBE_CA_FILE}")"
+    echo "${KUBE_CA_CRT}" | base64 -d > "${KUBE_CA_FILE}"
+    chmod 0600 "${KUBE_CA_FILE}"
+}
+
+configureSecureTLSBootstrap() {
+    SECURE_TLS_BOOTSTRAPPING_DROP_IN="/etc/systemd/system/secure-tls-bootstrap.service.d/10-securetlsbootstrap.conf"
+    touch "${SECURE_TLS_BOOTSTRAPPING_DROP_IN}"
+    chmod 0600 "${SECURE_TLS_BOOTSTRAPPING_DROP_IN}"
+
+    tee "${SECURE_TLS_BOOTSTRAPPING_DROP_IN}" > /dev/null <<EOF
```

Comment:
```suggestion
    cat "${SECURE_TLS_BOOTSTRAPPING_DROP_IN}" <<EOF
```

no need to tee and drop the output; just use cat.